Week3/homework/step2-2.js

```diff
@@ -2,19 +2,28 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  console.log(numbers);
+  for (let j = 0; j < numbers.length; j++) {
+    if (numbers[j] % 3 === 0 && numbers[j] % 5 === 0) {
+      threeCallback();
+      fiveCallback();
+    } else if (numbers[j] % 3 === 0) {
+      threeCallback();
+    } else if (numbers[j] % 5 === 0) {
+      fiveCallback();
+    }
+  }
 }
 
-function sayThree(number) {
-  // Replace this comment and the next line with your code
-  console.log(number);
+function sayThree() {
+  console.log(3);
```

Comment:
You could go a bit further here, what if this function accepted the number as a parameter? You could then use that in the console log so that it's a bit more descriptive.